examples/arc.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python3
+#
+# Test for a push-button monome arc
+
+import asyncio
+import monome
+
+class ExampleArc(monome.Arc):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
this class is functionally identical to the Arc superclass, why is it declared here?